src/selectors/resourceSelectors.js

```diff
@@ -57,7 +57,12 @@ export const resourceHasChangesSinceLastSave = (state) => {
   if (lastSaveChecksum === undefined) {
     return true
   }
-  const rdf = new GraphBuilder(state.selectorReducer).graph.toCanonical()
-  const resourceChecksum = generateMD5(rdf)
-  return lastSaveChecksum !== resourceChecksum
+  // This is temporary until the setting of lastSaveChecksum is fixed.
```

Comment:
is the problem referred to here ticketed?

Reply:
Being fixed in https://github.com/LD4P/sinopia_editor/pull/1111